server/src/main/java/game/shooting/GameShootingPart.java

```diff
@@ -0,0 +1,53 @@
+package game.shooting;
+
+import game.shooting.matchers.PlayerBoardMatcher;
+import game.shooting.matchers.PlayerToShotSupplierMatcher;
+import model.preparing.Player;
+import model.shooting.board.ShootingBoard;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameShootingPart implements ObservableGame {
+    private List<GameObserver> observers;
+    private Judge judge;
+    private PlayerToShotSupplierMatcher shotSupplier;
+    private final PlayerBoardMatcher playerBoardMatcher;
+
+    private GameShootingPart(Judge judge, PlayerToShotSupplierMatcher hitSupplier,
+                             PlayerBoardMatcher playerBoardMatcher) {
+        observers = new ArrayList<>();
+        this.judge = judge;
+        this.shotSupplier = hitSupplier;
+        this.playerBoardMatcher = playerBoardMatcher;
+    }
+
+    public static GameShootingPart create(Judge judge, PlayerBoardMatcher playerBoardMatcher,
```

Comment:
Is there a need to declare it as static? All fields are already in this class provided by constructor. Nothing  use this method so far, but it is already public and static

Reply:
It's a factory method - constructor is private